poc_iot_verifier/src/traits/gateway_info_resolver.rs

```diff
@@ -0,0 +1,8 @@
+use crate::{gateway_resp::FollowerGatewayResp, Result};
+use async_trait::async_trait;
+use helium_crypto::PublicKey;
+
+#[async_trait]
```

Comment:
You should put the implementation for the follower service along side the gatewayinforesolver trait. Additionally, there's no reason to put these traits inside their own "traits" module. Move this module into the root. (You wouldn't have a "structs" module for example)

Reply:
> You should put the implementation for the follower service along side the gatewayinforesolver trait.

I updated this in 656f9f4

> Additionally, there's no reason to put these traits inside their own "traits" module. Move this module into the root. (You wouldn't have a "structs" module for example)

We should probably do another pass for this, not in this PR since that is happening in several other places .